builtin/providers/fastly/resource_fastly_service_v1.go

```diff
@@ -854,6 +888,73 @@ func resourceServiceV1Update(d *schema.ResourceData, meta interface{}) error {
 			}
 		}
 
+		// Find differences in VCLs
+		if d.HasChange("vcl") {
+			// Note: as above with Gzip and S3 logging, we don't utilize the PUT
+			// endpoint to update a VCL, we simply destroy it and create a new one.
+			oldVCLVal, newVCLVal := d.GetChange("vcl")
+			if oldVCLVal == nil {
+				oldVCLVal = new(schema.Set)
+			}
+			if newVCLVal == nil {
+				newVCLVal = new(schema.Set)
+			}
+
+			oldVCLSet := oldVCLVal.(*schema.Set)
+			newVCLSet := newVCLVal.(*schema.Set)
+
+			remove := oldVCLSet.Difference(newVCLSet).List()
+			add := newVCLSet.Difference(oldVCLSet).List()
+			_ = add
```

Comment:
Oops, that was just debugging cruft leftover. Assigning to `_` lets you get around `add declared and not used` errors while you're in the middle of things.